crates/paths/src/paths.rs

```diff
@@ -189,6 +189,12 @@ pub fn themes_dir() -> &'static PathBuf {
     THEMES_DIR.get_or_init(|| config_dir().join("themes"))
 }
 
+/// Returns the path to the snippets directory.
+pub fn snippets_dir() -> &'static PathBuf {
+    static THEMES_DIR: OnceLock<PathBuf> = OnceLock::new();
```

Comment:
```suggestion
    static SNIPPET_DIR: OnceLock<PathBuf> = OnceLock::new();
```
We could also use `LazyLock` here maybe.